modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,46 @@
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+
+With {pipelines-shortname}, to run pods in the user namespace, you must pass the annotations that CRIO expects. To add these annotations for all users, you use the `oc edit clustertask buildah` command to edit the `buildah` cluster task. Or, to add the annotations to a specific namespace, you export the cluster task as as task to that namespace.
+
+.Procedure
+
+. Use `tkn create` feature to automate <finish this phrase>.
++
+[source,terminal,subs="attributes+"]
```

Comment:
Per the OCP doc guidlines, for multiple CLI commands, you must include only one command per code block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output. So this code block and all the other ones in this PR need to be broken down so that each command uses its own code block.